### ISSUE TITLE
Upload customized drop label issues

### ISSUE THREAD

**Issue by rolfsmeds** [vaadin/docs#2615]
Change the customization sample to not contain important text, and add a note that the label is not visible to screen readers so any text that all users should we aware of needs to be placed elsewhere.
---

If you think this issue is important, add a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to help the community and maintainers prioritize this issue.
